packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -38,6 +38,7 @@ export const ButtonView: IButtonComponent['view'] = props => {
       disabled={disabled}
       aria-disabled={disabled}
       aria-label={ariaLabel}
+      ref={buttonRef}
```

Comment:
I find it just a little odd that Button doesn't have a button slot and that it's the only one that actually uses ref while the rest of the button variants do have a button slot and they all use componentRef. Not really an issue so much as me being a little confused.

Reply:
The way slots work is that you compose a component via slots of different components. `Button` doesn't have a `button` slot because you are inherently composing it. `MenuButton` and `SplitButton` do have `button` slots because we are using the vanilla `Button` we built as a slot to compose them.

We use `ref` on the `Button` because we are referencing the DOM `button`, whereas we cannot use `ref` on `MenuButton` or `SplitButton` because we are referencing our custom-built `Button` that does not have `ref` because it is a stateless component. To go around this we leverage `componentRef` that is extended from `IBaseProps` into our component props.